data/builtInFeatures.js

```diff
@@ -0,0 +1,120 @@
+// https://github.com/zloirock/core-js
+
+/* eslint-disable quotes */
+module.exports = {
+  // es2015
+  // core-js/fn/map
+
+  // "es6.typed/array-buffer": "typed arrays / ",
```

Comment:
I think these would be covered with all the `int8-array` ones below

typed arrays / ArrayBuffer[Symbol.species]
typed arrays / constructors require new
typed arrays / constructors accept generic iterables
typed arrays / correct prototype chains
typed arrays / %TypedArray%.from
typed arrays / %TypedArray%.of
typed arrays / %TypedArray%.prototype.subarray
typed arrays / %TypedArray%.prototype.join
typed arrays / %TypedArray%.prototype.indexOf
typed arrays / %TypedArray%.prototype.lastIndexOf
typed arrays / %TypedArray%.prototype.slice
typed arrays / %TypedArray%.prototype.every
typed arrays / %TypedArray%.prototype.filter
typed arrays / %TypedArray%.prototype.forEach
typed arrays / %TypedArray%.prototype.map
typed arrays / %TypedArray%.prototype.reduce
typed arrays / %TypedArray%.prototype.reduceRight
typed arrays / %TypedArray%.prototype.reverse
typed arrays / %TypedArray%.prototype.some
typed arrays / %TypedArray%.prototype.sort
typed arrays / %TypedArray%.prototype.copyWithin
typed arrays / %TypedArray%.prototype.find
typed arrays / %TypedArray%.prototype.findIndex
typed arrays / %TypedArray%.prototype.fill
typed arrays / %TypedArray%.prototype.keys
typed arrays / %TypedArray%.prototype.values
typed arrays / %TypedArray%.prototype.entries
typed arrays / %TypedArray%.prototype[Symbol.iterator]
typed arrays / %TypedArray%[Symbol.species]